src/components/Modal/QRCode.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import { qrcode } from "pure-svg-code";
+  import Modal from "@app/Modal.svelte";
+  import { state } from "@app/session";
+  import type { Config } from "@app/config";
+
+  export let uri: string;
+  export let config: Config;
+
+  $: svgString = qrcode({
+    content: uri,
+    width: 225,
+    height: 225,
+    color: "black",
+    background: "white",
+    ecl: "M",
+  });
+
+  const onClickConnect = () => {
+    state.connectMetamask(config);
+  };
+</script>
+
+<style>
+  .qrcode-wrapper {
+    width: fit-content;
```

Comment:
```suggestion
    width: min-content;
```
Fixes the firefox issue

Reply:
oh nice